test/test_controller.py

```diff
@@ -230,16 +231,41 @@ def test_email_verification_activates_a_respondent(self, _, mock_notify):
         # And the respondent state is CREATED
         db_respondent = respondents()[0]
         self.assertEqual(db_respondent.status, RespondentStatus.CREATED)
-
         # When the email is verified
         frontstage_url = mock_notify.call_args[0][2]
         token = frontstage_url.split('/')[-1]
         self.put_email_verification(token, 200)
-
         # Then the respondent state is ACTIVE
         db_respondent = respondents()[0]
         self.assertEqual(db_respondent.status, RespondentStatus.ACTIVE)
 
+    # This tests that the email verification URL is dynamically taken from a parameter in the configuration called
```

Comment:
Good naming of the test and the Given, When, Then structure should mean we don't need any further comments